### PR TITLE
Documentation for the 'break' keyword in the std library

### PR DESCRIPTION
**Pull request by Deicyde**

### CODE REVIEW

**Comment by JohnCSimon** [comment]
Ping from triage
@Kimundi  Can you please review this PR? 

Thanks.

**Comment by JohnCSimon** [comment]
Ping from triage again
@Kimundi Can you please review this PR?

Thanks.

**Comment by JohnCSimon** [comment]
Ping from triage
@SimonSapin Can you please review this PR? 
Thanks.

**Comment by JohnTitor** [comment]
Ping from triage: @SimonSapin could you review it?

File: src/libstd/keyword_docs.rs [hunk 1]
```diff
@@ -32,10 +32,55 @@ mod as_keyword { }
...
+///     println!("In the outer loop!");
+///     'inner: loop {
+///         println!("In the inner loop!");
+///         break 'outer;
+///     }
+///     unreachable!();
+/// }
+/// ```
+/// 
+/// `break` can also be to return a value from a `loop`, `for` or `while` block. To ensure type
```
Comment (SimonSapin) [inline src/libstd/keyword_docs.rs]: “can also be *used* to return a value”?

File: src/libstd/keyword_docs.rs [hunk 2]
```diff
@@ -32,10 +32,55 @@ mod as_keyword { }
...
+/// safety, all returned values must be of the same type.
```
Comment (SimonSapin) [inline src/libstd/keyword_docs.rs]: Maybe add something like “If `break` is used more than once for a given loop,”?

**Review by SimonSapin — APPROVED** [review]

**Comment by Alexendoo** [comment]
Ping from triage: Hi @Deicyde, have you had a chance to look at the recent review?

Also the [build failure](https://github.com/rust-lang/rust/pull/64270#issuecomment-529152898) is caused by trailing whitespace in the comments, if you could remove that that it would be great thanks.

**Comment by Dylan-DPC** [comment]
Hey @Deicyde, I didn't realise this pr exists when I was reviewing https://github.com/rust-lang/rust/pull/65544. So I am closing this because the other one will be merged soon. Sorry about it and thanks for taking the time to contribute to the language. :)

**Pull request closed without merging**